scss/atoms/typography/_mixins.scss

```diff
@@ -0,0 +1,26 @@
+// ==========================================================================
+// Typography functions
+// ==========================================================================
+
+// Adds a single pixel underline (so nice) on retina
+// Usage: `@include slime-underline;`
```

Comment:
"slime" [sic]

Also, the "reimplementing underlines" technique in general seems (a) overcomplicated/overkill (b) not easy to override